### DISCUSSION TITLE
Why not releasing more frequently?

### DISCUSSION THREAD

**Discussion by yuhr** [software-mansion/react-native-screens#2066] [Q&A] [answered]
I think every single PR merge should ideally cause a release (I don't say *must*, but definitely *should*). What is stopping you from new releases for months?

**Comment by tboba** [comment, marked as answer]
Hi @yuhr, it's not that we're blocked from releasing Screens, but there are a couple of reasons why sometimes we need to wait with the release. Sometimes it's because we want to release a new version along with third-party packages (e.g. react-native-reanimated), but the main reason is that we don't want to lose the distinction between more and less important features being introduced (that's why we also don't want to release after every commit to main). 

However, I agree that we should release the package more frequently - the next release is planned to be released in the next week, but we have in mind to release the package more often!
For now I'm closing this discussion, but feel free to comment or discuss here your thoughts if you have any!
